docs/abi-spec.rst

```diff
@@ -536,18 +572,35 @@ An event description is a JSON object with fairly similar fields:
 
 - ``anonymous``: ``true`` if the event was declared as ``anonymous``.
 
+Errors look as follows:
+
+- ``type``: always ``"error"``
+- ``name``: the name of the error.
+- ``inputs``: an array of objects, each of which contains:
+
+  * ``name``: the name of the parameter.
+  * ``type``: the canonical type of the parameter (more below).
+  * ``components``: used for tuple types (more below).
+
+.. note::
+  There can be multiple errors with the same name and even with identical signature
```

Comment:
Maybe we should add a note here with something like "note: errors - even if defined in different contracts/libraries - are placed in the same (global) namespace". We should also add the same note in the event section.

Reply:
I added some more text.